server/migrations/20181107082847-create-rating.js

```diff
@@ -0,0 +1,41 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Ratings', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    userId: {
+      type: Sequelize.INTEGER,
+      onDelete: 'CASCADE',
+      allowNull: false,
+      references: {
+        model: 'Users',
+        key: 'id'
+      }
+    },
+    articleId: {
+      type: Sequelize.INTEGER,
+      onDelete: 'CASCADE',
+      allowNull: false,
+      references: {
+        model: 'Articles',
+        key: 'id'
+      }
+    },
+    rating: {
```

Comment:
One more thing. Do a max and min value for ratings.

Reply:
I have added the max and min values for rating.